torch/nn/utils/weight_norm.py

```diff
@@ -113,9 +110,10 @@ def remove_weight_norm(module, name='weight'):
         >>> m = weight_norm(nn.Linear(20, 40))
         >>> remove_weight_norm(m)
     """
-    for hook in module._forward_pre_hooks.values():
+    for k, hook in module._forward_pre_hooks.items():
         if isinstance(hook, WeightNorm) and hook.name == name:
             hook.remove(module)
+            del module._forward_pre_hooks[k]
```

Comment:
No, not anymore. Previously, `weight_norm` held on to the handle, so remove could just call `handle.remove()`. However, this broke pickling in some earlier versions of 2.7, so `WeightNorm.remove()` just unapplies the transformation and the hook is deleted here.